scoring/score_submission.py

```diff
@@ -1,11 +1,16 @@
+import operator
 import os
 
 from absl import app
 from absl import flags
 from absl import logging
+import numpy as np
+import pandas as pd
 import scoring_utils
+from tabulate import tabulate
 
-from scoring import scoring
+from scoring import performance_profile
+from scoring.performance_profile import check_if_minimized
```

Comment:
Doesn't have to be explicitly imported since `performance_profile` is already imported anyway. Alternatively, all needed function from `performance_profile` could be imported explicitly.